host/Tupfile

```diff
@@ -2,6 +2,5 @@ include_rules
 : |> sed 's/{{TUF-ROOT-KEYS}}/@(TUF_ROOT_KEYS)/g' cli/root_keys.go.tmpl > %o |> cli/root_keys.go
 : cli/root_keys.go |> !cgo |> bin/flynn-host
 : bin/flynn-host |> gzip -9 --keep bin/flynn-host |> bin/flynn-host.gz
-: |> !go ./flynn-init |> bin/flynn-init
+: |> !cgo ./flynn-init |> bin/flynn-init
```

Comment:
This will break containerinit if the rootfs doesn't have libc. Perhaps we should put the libcontainer part in `flynn-host` instead?